app/build.gradle

```diff
@@ -45,11 +45,19 @@ android {
         debug {
             applicationIdSuffix ".debug"
             pseudoLocalesEnabled false
+            manifestPlaceholders = [
```

Comment:
We can use manifest placeholders to change the app icon based on the build version. This is done to replace the need of maintaining a separate folder in the source.

Reply:
I'm no longer seeing the blue icon for DEBUG builds. Any thoughts why?

Pixel 2; Android R

Reply:
This is now fixed, thanks for noticing!